delocate/tools.py

```diff
@@ -320,6 +322,15 @@ def get_rpaths(filename):
         assert cmdsize.startswith('cmdsize ')
         paths.append(RPATH_RE.match(path).groups()[0])
         line_no += 2
+    # We'll search the extra library paths in a specific order:
```

Comment:
I'd prefer to have the `get_rpaths` function return the rpaths from the lib, and then, in `tree_libs`, concatenate the results from here, with the results from the environment variables.  OK with you?

Reply:
Yeah, no prob.